artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -253,6 +255,8 @@
 
    private Boolean enableMetrics = null;
 
+   private Integer managementMessageAttributeSizeLimit = AddressSettings.MANAGEMENT_MESSAGE_ATTRIBUTE_SIZE_LIMIT;
```

Comment:
Should be null. So things like merge of address settings dont merge unset values.

Reply:
ok, that makes sense. thanks